### ISSUE TITLE
[Bug]: Not one single video is playing

### ISSUE THREAD

**Issue by C-O-D** [MarmadileManteater/FreeTubeCordova#226]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

I can't get a single video to run here strangely enough. The player loads and loads and loads no matter what API, instance or format I set, none of these problems however with the original freetube...

### Expected Behavior

I click on play and the video is played...

### Issue Labels

content not loading

### FreeTube Version

0.18.0.107 Beta

### Operating System Version

Android 13, MIUI 14.0.2

### Installation Method

.apk

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by MarmadileManteater** [comment]
Regardless of your API choice, Invidious proxying is always enabled in FreeTube Cordova because CORS works differently than in `electron` (see #5), and so this is a known issue with YT IP blocking invidious instances from proxying video playback endpoints.  See https://github.com/FreeTubeApp/FreeTube/issues/3892 and https://github.com/iv-org/invidious/issues/4045 for more information. 

It isn't every single instance, but it is absolutely most of them. The instance I get the best results with is `yt.artemislena.eu`, but that could obviously change if the traffic to this instance increased enough for YouTube to IP block them.

Personally, I actually do not watch videos through the video js player interface because of the poor performance of proxying videos, and instead, lately, I've been opting for just opening the download links in my preferred browser and streaming the video directly through there. There's no dash or anything fancy, but for my purposes it is enough for now, and I want to change it, and I have been working on changing it in the past, but local api streaming has eluded me for quite some time.

TLDR: closing because this ticket is already eclipsed by #5 and the upstream ticket FreeTubeApp#3892.
